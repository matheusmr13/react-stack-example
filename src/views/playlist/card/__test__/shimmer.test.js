import React from 'react';
import { shallow } from 'enzyme';
import PlaylistCardShimmer from 'views/playlist/card/shimmer';

describe('PlaylistCardShimmer component', () => {
	it('should render properly', () => {
		const playlistCard = shallow(<PlaylistCardShimmer />);
		expect(playlistCard).toMatchSnapshot();
	});
});
