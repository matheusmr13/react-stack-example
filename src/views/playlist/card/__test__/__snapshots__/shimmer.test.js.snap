// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaylistCardShimmer component should render properly 1`] = `
<ContentLoader
  className=""
  height={337}
  preserveAspectRatio="xMidYMid meet"
  primaryColor="#282828"
  secondaryColor="#404040"
  speed={2}
  type="facebook"
  width={300}
>
  <rect
    height="298"
    rx="5"
    ry="5"
    width="298"
    x="1"
    y="1"
  />
  <rect
    height="30"
    rx="5"
    ry="5"
    width="200"
    x="50"
    y="310"
  />
</ContentLoader>
`;
